specification/voiceservices/data-plane/Microsoft.VoiceServices/stable/2023-10-01/swagger.json

```diff
@@ -0,0 +1,1093 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft.VoiceServices.ACG.ProvisioningPlatform.API",
+    "version": "2023-10-01"
+  },
+  "securityDefinitions": {
+    "OAuth2": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "flow": "application",
+      "tokenUrl": "https://login.microsoftonline.com/common/oauth2/v2.0/token",
```

Comment:
I believe our service uses a tenant specific authentication endpoint for this flow. How do we express that here?

Reply:
Our flow looks like this, https://learn.microsoft.com/en-us/entra/identity-platform/v2-oauth2-client-creds-grant-flow#get-a-token but as far as I can tell CI rejects any templating, and I've been unable to find any docs on how this is meant to be done.

Reply:
I think the SDK generators all have ways to provide more specific information on authentication, so what you have here should be fine.

Reply:
Thanks - that makes sense.